pkg/endpoint/endpoint.go

```diff
@@ -90,7 +90,7 @@ type Connector interface {
 	GenerateRequest(config *ZoneConfiguration, req *certificate.Request) (err error)
 	// ResetCertificate resets the state of a TPP certificate.
 	// This function is idempotent, i.e., it won't fail if there is nothing to be reset.
-	// It returns an error of type *errCertNotFound if the certificate is not found.
+	// It returns an error of type *tpp.ErrCertNotFound if the certificate is not found.
```

Comment:
This change in the doc is not correct given this is the endpoint interface which is generic and indepent of the platform which VCert is connecting and then we can't refer here to specific elements to tpp platform

Reply:
@marcos-albornoz Yes, currently `ResetCertificate` is not supported for VaaS. It's TPP only (also, that structure only lives on TPP's side of things for now):
https://github.com/Venafi/vcert/blob/master/pkg/venafi/cloud/connector.go#L843-L845

Reply:
Resolved by latest changes